### PR TITLE
Morse code

### PR DESCRIPTION
**Pull request by johnie261**
Hi there,
### @banzaman and I did the following:

- Created a method to decode a Morse code character.
- Created a method to decode an entire word in Morse code.
- Created a method to decode the entire message in Morse code.

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @johnie261 and @banzaman , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have all the main headers of the Readme [ template](https://github.com/microverseinc/readme-template) such as future features

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,132 @@
...
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (christianbiring1) [inline README.md:108]: [OPTIONAL] 
- [x] Always remember to provide a correct link that is not broken up for better guidance while navigating through your repos.

**Review by christianbiring1 — APPROVED** [review]
Hi @johnie261 @banzaman , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [x] All the linters checks are green :heavy_check_mark: 
- [x] Descriptive R\PR `title` and `summary` :
- [ ] The meassge is decoded correctly :+1:
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by banzaman** [comment]
Thanks for you time and advise @christianbiring1

**Pull request merged**